modules/dnn/src/op_inf_engine.hpp

```diff
@@ -98,11 +99,16 @@ class InfEngineBackendNet
 private:
     InferenceEngine::Builder::Network netBuilder;
 
-    InferenceEngine::InferenceEnginePluginPtr enginePtr;
-    InferenceEngine::InferencePlugin plugin;
     InferenceEngine::ExecutableNetwork netExec;
     InferenceEngine::BlobMap allBlobs;
-    InferenceEngine::TargetDevice targetDevice;
+    std::string device_name;
+#if INF_ENGINE_VER_MAJOR_LE(2019010000)
+    InferenceEngine::InferenceEnginePluginPtr enginePtr;
+    InferenceEngine::InferencePlugin plugin;
+#else
+    InferenceEngine::Core core;
```

Comment:
If there are two networks withe their own `InferenceEngine::Core` will they share the plugins? I mean can we create two networks in this way on Myriad, in example?